zone/mob.cpp

```diff
@@ -512,6 +512,18 @@ Mob::Mob(
 	mob_close_scan_timer.Trigger();
 
 	SetCanOpenDoors(true);
+
+	is_boat = (
+        race == RACE_SHIP_72 ||
+        race == RACE_LAUNCH_73 ||
+        race == RACE_GHOST_SHIP_114 ||
+        race == RACE_SHIP_404 ||
+        race == RACE_MERCHANT_SHIP_550 ||
+        race == RACE_PIRATE_SHIP_551 ||
+        race == RACE_GHOST_SHIP_552 ||
+        race == RACE_BOAT_533
+    );
```

Comment:
Formatting.

Reply:
I think we still keep `IsBoat()` with the or checks, it served a purpose that's worth keeping

The getter to the bool can just be `GetIsBoat()` with `SetIsBoat(bool)`

Reply:
Uncalled?  Or leave all the other spots using that instead of the variable?

Reply:
Let me know if this is what you wanted.